cl/recap/mergers.py

```diff
@@ -1508,13 +1514,38 @@ def merge_attachment_page_data(
                 )
             except ValueError:
                 pass
-        rd.save()
+        await rd.asave()
 
         # Do *not* do this async — that can cause race conditions.
-        add_items_to_solr([rd.pk], "search.RECAPDocument")
+        await sync_to_async(add_items_to_solr)([rd.pk], "search.RECAPDocument")
+
+    de_query = RECAPDocument.objects.filter(
+        docket_entry=de,
+        document_number=document_number,
+    )
 
-    mark_ia_upload_needed(de.docket, save_docket=True)
-    process_orphan_documents(
+    dupe_doc_ids = (
+        de_query.values("pacer_doc_id")
+        .filter()
```

Comment:
Do you need this filter? I think you can remove it, right?

Reply:
removed